ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -1025,26 +1025,26 @@ presubmits:
     always_run: true
     branches:
     - master
-    cluster: vsphere
-    context: ci/prow/e2e-vsphere
+    cluster: api.ci
+    context: ci/prow/e2e-vsphere-ovn
```

Comment:
that `cluster` change looks wrong? autogenerated thing that needs to be un-autogenerated maybe?

Reply:
you are right there is nothing that scheduled the test for vsphere automatically. Changed it